lib/src/hidable_controller.dart

```diff
@@ -70,56 +62,34 @@ typedef HidableVisibility = double Function(
 /// And the [sizeNotifier] for providing/updating the hideable status.
 class HidableController {
   ScrollController scrollController;
-  double size;
   HidableVisibility? hideableVisibility;
+  double deltaFactor;
 
   HidableController({
     required this.scrollController,
-    required this.size,
     this.hideableVisibility,
+    this.deltaFactor = 0.08,
```

Comment:
I tinkered with the delta and 0.08 seemed like too fast for me, I set it to 0.04 after my issue message.

Reply:
Cool, let me test. If seems fine I'll make the default 0.04 also. But I think, the current one works fine regarding speed.

Reply:
@radeonovich gonna go with 0.06, 4 seemd very slow and 8 was fast right. I tested with the 0.06 and it seems ok.